guides/source/form_helpers.md

```diff
@@ -364,166 +334,85 @@ IMPORTANT: All forms using `form_with` implement `remote: true` by default. Thes
-<option value="3" data-size="3.4 million">Berlin</option>
+<select name="city" id="city">
+  <option value="BE">Berlin</option>
+  <option value="LX">Lisbon</option>
+  <option value="MD">Madrid</option>
+</select>
 ```
 
-### Select Boxes for Dealing with Model Objects
+This way, the user will see the full city name, but `params[:city]` will be one of `"BE"`, `"LX"`, or `"MD"`.
 
-In most cases form controls will be tied to a specific model and as you might expect Rails provides helpers tailored for that purpose. Consistent with other form helpers, when dealing with a model object drop the `_tag` suffix from `select_tag`:
-
-If your controller has defined `@person` and that person's city_id is 2:
-
-```ruby
-@person = Person.new(city_id: 2)
-```
+Finally, we can specify a default choice for the select box with the `:selected` argument:
 
 ```erb
-<%= select(:person, :city_id, [['Lisbon', 1], ['Madrid', 2], ['Berlin', 3]]) %>
+<%= form.select :city, [["Berlin", "BE"], ["Lisbon", "LX"], ["Madrid", "MD"]], selected: "LX" %>
 ```
 
-will produce output similar to
+Output:
 
 ```html
-<select name="person[city_id]" id="person_city_id">
-  <option value="1">Lisbon</option>
-  <option value="2" selected="selected">Madrid</option>
-  <option value="3">Berlin</option>
+<select name="city" id="city">
+  <option value="BE">Berlin</option>
+  <option value="LX" selected="selected">Lisbon</option>
+  <option value="MD">Madrid</option>
 </select>
 ```
 
-Notice that the third parameter, the options array, is the same kind of argument you pass to `options_for_select`. One advantage here is that you don't have to worry about pre-selecting the correct city if the user already has one - Rails will do this for you by reading from the `@person.city_id` attribute.
+### Select Boxes and Model Objects
 
-As with other helpers, if you were to use the `select` helper on a form builder scoped to the `@person` object, the syntax would be:
+Like other form controls, a select box can be bound to a model attribute. For example, if we have a `@person` model object like:
 
-```erb
-<%= form_with model: @person do |person_form| %>
-  <%= person_form.select(:city_id, [['Lisbon', 1], ['Madrid', 2], ['Berlin', 3]]) %>
-<% end %>
+```ruby
+@person = Person.new(city: "MD")
 ```
 
-You can also pass a block to `select` helper:
+The following form:
 
 ```erb
-<%= form_with model: @person do |person_form| %>
-  <%= person_form.select(:city_id) do %>
-    <% [['Lisbon', 1], ['Madrid', 2], ['Berlin', 3]].each do |c| %>
-      <%= content_tag(:option, c.first, value: c.last) %>
-    <% end %>
-  <% end %>
+<%= form_with model: @person do |form| %>
+  <%= form.select :city, [["Berlin", "BE"], ["Lisbon", "LX"], ["Madrid", "MD"]] %>
 <% end %>
 ```
 
-WARNING: If you are using `select` or similar helpers to set a `belongs_to` association you must pass the name of the foreign key (in the example above `city_id`), not the name of association itself.
```

Comment:
I think this was worth retaining - the new docs don't talk about associations at all, which is a common use case for `f.select`.